src/marquee/Marquee.style.tsx

```diff
@@ -54,4 +56,10 @@ export const StyledMarqueeItem = styled.li<{
   animation-duration: ${props => `${animationDuration * props.totalItems}ms`};
   animation-iteration-count: infinite;
   animation-delay: ${props => `${animationDuration * props.position}ms`};
+  width: calc(100% - (${horizontalSpace.outer}*2));
+  /* cannot use normalizeHorizontally directly here because it conflicts with props */
```

Comment:
nit: use `&&` to increase specificity and thus remove !**important**

```suggestion
&& {
 padding-left: ${horizontalSpace.inner};
  padding-right: ${horizontalSpace.inner};
  margin-left: ${horizontalSpace.outer};
  margin-right: ${horizontalSpace.outer};
}
```

Reply:
`!important` here is copy/pasted from https://github.com/blablacar/ui-library/blob/master/src/layout/layoutNormalizer.tsx#L18

So I'm not sure duplicating the class is really better than "staying coherent"

I don't have a strong opinion so I'll wait for @wakooka's review if possible

Reply:
I don't understand the comment "it conflict with props", what was the issue?

Reply:
It wouldn't pass the linter since it's using it's own props that is not in the declared props for this component.
<img width="1420" alt="Screen Shot 2021-02-05 at 10 21 02" src="https://user-images.githubusercontent.com/373381/107017540-8c863a00-679f-11eb-987e-e9e949b66e07.png">


But for some reason I stopped there while I could simply add the missing prop. I've just updated the PR, it's now working fine with `normalizeHorizontally`